plugins/source/gcp/client/client.go

```diff
@@ -88,6 +88,9 @@ func (c *Client) ID() string {
 	if c.OrgId != "" {
 		return "org:" + c.OrgId
 	}
+	if c.Location != "" {
+		return "project:" + c.ProjectId + ":location:" + c.Location
+	}
 	return c.ProjectId
```

Comment:
Nit: Can we do that the last option will be `project:` + c.ProjectId so at least we only have two types of IDs

Reply:
Sure, done in https://github.com/cloudquery/cloudquery/pull/6807/commits/3dff1260b044228bd2cc9c43d51f046c10c75904